packages/core/src/item.ts

```diff
@@ -278,6 +278,28 @@ export interface AuditResult {
     fieldIndex: number;
 }
 
+export class ItemHistory extends Serializable {
+    constructor(vals: Partial<ItemHistory> = {}) {
+        super();
+        Object.assign(this, vals);
+    }
+
+    date: Date = new Date();
+
+    updatedBy: AccountID = "";
+
+    name?: string;
```

Comment:
All class properties require an initialiser to be able to deserialised properly, even if they're optional. This is probably why these weren't stored. I.e. this needs to be:

```
name?: string = undefined;
```

Reply:
Ah, TIL!